code/modules/hydroponics/grown/melon.dm

```diff
@@ -12,7 +12,7 @@
 	growing_icon = 'icons/obj/hydroponics/growing_fruits.dmi'
 	icon_dead = "watermelon-dead"
 	genes = list(/datum/plant_gene/trait/repeated_harvest)
-	mutatelist = list(/obj/item/seeds/watermelon/holy)
+	mutatelist = list(/obj/item/seeds/watermelon/holy, /obj/item/seeds/watermelon/barrel)
```

Comment:
this is actually a balance concern, as holy melons are something that botany pumps out against many magical threats

giving watermelons a 50% chance to turn into a "haha funny alcohol" plant instead of a "you can now deconvert cultists without the aid of a chaplain or spending thousands of dollars at cargo, and also you don't get hardstunned by cultists in a single click anymore" plant when they mutate sounds like it'd lead to a lot of, ah, "frustration".

Reply:
feature

Reply:
I'd suggest making them a plump helmet or pumpkin mutation instead

Reply:
plump helmets arent watermelons

Reply:
Balance concerns? In my botany? Get outta here.

Reply:
Holymelons are too easy to get imo

Reply:
> this is actually a balance concern, as holy melons are something that botany is expected to pump out when magical threats to the station are called out.

I don't think this applies to new botany, since you can just pump the instability while having autogrow on, harvest and then plant a few trays in order to get all mutations of that plant.

Reply:
> I don't think this applies to new botany, since you can just pump the instability while having autogrow on, harvest and then plant a few trays in order to get all mutations of that plant.

New botany, as fun as it is, is severely size-restricted. Planting seeds with inherent 100 instability does not guarantee that they will instantly mutate, making the process time-consuming and frustrating. 

Compared to virology, xenobiology and genetics, other professions where one is trying to find certain objectives via random mutations, botany is much slower and the sample sizes are much smaller. These two facts combined make hunting for plant mutations much more frustrating, as low-number high-uncertainty leads to very spread out distribution of results. One shift your first tomato goes blue, the next shift you get unlucky and even after 8 you only have flesh tomatos. This is inherent in the low number statistics.

This is not unique to this PR, but I would try and limit the amount of branching paths.

Reply:
> Holymelons are too easy to get imo

if need be, you could make watermelons (only) mutate into barrel plants and make barrel plants mutate into holy melons

just please don't try to balance this stuff by adding more botany RNG